application.py

```diff
@@ -160,6 +160,14 @@ def display():
     else: 
         return render_template('search.html')
 
+@app.route("/book",methods=['POST','GET'])
+@app.route("/book/<string:args>", methods= ['POST','GET'])
+def book(args=None):
+    data=db.query(Book).filter(Book.isbn.like(args))
```

Comment:
like operator will return all possible matches(even more than 1 result , if the isbn is provided incorectly), I suggest you to match the exact result with " = "

Reply:
1. If the user typed in only part of a title, ISBN, or author name, your search page should find matches for those as well! is their in lms.
so used like operator.